meta-resin-common/recipes-support/resin-init/resin-init-flasher-board.bb

```diff
@@ -0,0 +1,18 @@
+DESCRIPTION = "Board custom INIT file"
+SECTION = "console/utils"
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM = "file://${RESIN_COREBASE}/COPYING.Apache-2.0;md5=89aea4e17d99a7cacdbeed46a0096b10"
+
+PR = "r1"
```

Comment:
Remove as we need the default one which is r0. So we can just remove it completely for now.